README.md

```diff
@@ -1,4 +1,99 @@
-GEOPM SPANK PLUGIN FOR SLURM
-============================
-This repository maintains the spank geopm plugin to support the GEOPM
+GEOPM SPANK PLUGINS FOR SLURM
+=============================
+
+This repository maintains the spank plugins to support the GEOPM
 runtime with the SLURM resource manager.
+
+SETUP
+-----
+0. Install the geopmpolicy RPM on head and compute nodes.
+
+1. Create a /etc/geopm directory on the compute nodes.
+
+2. Create /etc/geopm/environment-default.json and/or
+/etc/geopm/environment-override.json on the compute nodes with the
+following contents:
+
+```
+    {
+        "GEOPM_POLICY": "/etc/geopm/node_policy.json",
+	"GEOPM_AGENT": "power_balancer"
+    }
+```
```

Comment:
out of date.  "GEOPM_POLICY" should be the shared memory, e.g. "/geopm_endpoint", and GEOPM_AGENT should not be set.